src/pages/recurring-invoices/common/hooks.tsx

```diff
@@ -653,19 +653,19 @@ export function useRecurringInvoiceFilters() {
     },
     {
       label: t('active'),
-      value: 'paid',
+      value: 'active',
       color: 'white',
```

Comment:
I missed this bug on custom filters for recurring invoices so I just changed this. I thought this was a bit much to create a new PR.